### DISCUSSION TITLE
For deletion.

### DISCUSSION THREAD

**Discussion by stark1tty** [sneakers-the-rat/paper-feeds#19] [Ideas]
Edit: Cannot delete this myself. Getting a better idea what is going on here. Could we please turn on the ability to delete discussions? :)

**Comment by roaldarbol** [comment]
I'll just close it no worries - always takes me a long time figuring out what projects are about and how they're implemented - glad to have you snooping around! 😃🚀

**Comment by sneakers-the-rat** [comment]
i had no idea you couldn't delete discussion posts. huh.
